debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -589,13 +589,16 @@ public static SchemaRefreshMode parse(String value) {
                                                       + "'table' (the default) each DB table will have a separate Kafka topic; "
                                                       + "'schema' there will be one Kafka topic per DB schema; events from multiple topics belonging to the same schema will be placed on the same topic");
 
+    /**
+     * @deprecated use {@link CommonConnectorConfig#SNAPSHOT_FETCH_SIZE} instead of
+     */
+    @Deprecated
```

Comment:
I added the following log message:
```
Property 'rows.fetch.size' will be removed in next releases, use property 'snapshot.fetch.size' instead of
```

Reply:
Cool; thanks!